tox.ini

```diff
@@ -1,38 +1,47 @@
 [tox]
-envlist = black,flake8,mypy,py{38,37,36,35,py27,py3,py}
-skipsdist = true
+envlist = 
+    black,flake8,import-order,mypy,manifest,
+    py{27,35,36,37,38,39-dev,py,py3}
+; requires = tox-conda
 
 [testenv]
+passenv = *
 setenv =
     PYTHONPATH = {toxinidir}
-deps =
-    .[test]
+install_command = python -m pip install --ignore-installed {opts} {packages}
+deps = -e.[test]
+whitelist_externals =
+    python
 commands =
-    pytest tests {posargs}
+    pip install -U setuptools
```

Comment:
Why is this necessary?

Reply:
Sometimes the tox env crash due to `easy-install.pth` not found on conda environment. This is a known issue and some solution steps can be found [here](https://github.com/ContinuumIO/anaconda-issues/issues/542). Not sure if it is due to conda or pip related error but updating setuptools does help.